test/onnx/dynamo/test_registry_dispatcher.py

```diff
@@ -191,304 +94,330 @@ def setUp(self):
             self.registry, self.diagnostic_context
         )
 
-        self.op_overload = torch.fx.Node(
+    @parameterized.expand(
+        [
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="aten::add.Tensor",
+                    op="call_function",
+                    target=torch.ops.aten.add.Tensor,  # type: ignore[attr-defined]
+                    args=(torch.tensor(3), torch.tensor(4)),
+                    kwargs={},
+                ),
+                ("aten", "add", "Tensor"),
+            ),
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="aten::sym_size",
+                    op="call_function",
+                    target=torch.ops.aten.sym_size,
+                    args=(),
+                    kwargs={},
+                ),
+                ("aten", "sym_size", None),
+            ),
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="builtin_add",
+                    op="call_function",
+                    target=operator.add,
+                    args=(1, 2),
+                    kwargs={},
+                ),
+                ("aten", "add", None),
+            ),
+        ]
+    )
+    def test_get_aten_name_on_supported_fx_node(self, node, expected_name):
```

Comment:
```suggestion
    def test_get_aten_name_on_supported_fx_node(self, _: str, node, expected_name):
```

I would use the first param as the name of the test. Then in each parameterization consider a name that shows what the case is testing. E.g. for the `"aten::add.Tensor"` case I guess we are testing overloads? `builtin_add` built-in functions (not aten)?

Reply:
Done